Formula/cosign.rb

```diff
@@ -0,0 +1,32 @@
+class Cosign < Formula
+  desc "Container Signing"
+  homepage "https://github.com/sigstore/cosign"
+  url "https://github.com/sigstore/cosign.git",
+    tag:      "v1.1.0",
+    revision: "67934a685ddc83aa7b0b8a55c911e299117afac5"
```

Comment:
```suggestion
      tag:      "v1.1.0",
      revision: "67934a685ddc83aa7b0b8a55c911e299117afac5"
```
Let's align the arguments to `url`.